pyodc/codec.py

```diff
@@ -151,6 +151,66 @@ def from_dataframe(cls, column_name: str, data: pd.Series, data_type: DataType,
     def decode(self, stream):
         return struct.pack("<d", self.min).split(b"\x00", 1)[0].decode("utf-8")
 
+class VariableLengthConstantString(Constant):
```

Comment:
This is a very long class name.

In the header, this will be converted directly into a string which is used to identify the codec. I note also that "Variable Constant" is a very odd combination in a name.

Can we suggest something cleaner.